### PR TITLE
Update display names and descriptions of OIDC Scopes

### PR DESCRIPTION
**Pull request by vihanga-liyanage**
Refers https://github.com/wso2-enterprise/asgardeo-product/issues/2169

### CODE REVIEW

File: features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml [hunk 1]
```diff
@@ -16,22 +16,22 @@
 ~ under the License.
 -->
 <Scopes>
-    <Scope id="openid">
+    <Scope id="openid" displayName="Open ID" description="Used to indicate the use of OpenID Connect protocol">
```
Comment (mefarazath) [inline features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml:19]:
```suggestion
    <Scope id="openid" displayName="Open ID" description="Indicate the use of OpenID Connect protocol">
```

File: features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml [hunk 2]
```diff
@@ -16,22 +16,22 @@
...
         <Claim>
             sub,email,email_verified,name,family_name,given_name,middle_name,nickname,preferred_username,upn,groups,profile,picture,website,gender,birthdate,zoneinfo,locale,updated_at,phone_number,phone_number_verified,address,street_address,country,formatted,postal_code,locality,region
         </Claim>
     </Scope>
-    <Scope id="email">
+    <Scope id="email" displayName="Email" description="View the e-mail address of the user.">
```
Comment (mefarazath) [inline features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml:24]: Do we need . at the end?

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java [hunk 3]
```diff
@@ -1904,10 +1904,21 @@ public static OauthTokenIssuer getOAuthTokenIssuerForOAuthApp(OAuthAppDO appDO)
...
+                if (StringUtils.isNotEmpty(displayName)) {
+                    scope.setDisplayName(displayName);
+                } else {
+                    scope.setDisplayName(configType);
+                }
+
+                String description = omElement.getAttributeValue(new QName("description"));
+                if (StringUtils.isNotEmpty(description)) {
+                    scope.setDescription(description);
+                }
```
Comment (mefarazath) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java:1920]: Handle empty description from config file

File: features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml (hunk 2 above)
Comment (vihanga-liyanage) [reply features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml:24]: for descriptions, we've used period.

File: features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml [hunk 4]
```diff
@@ -16,22 +16,22 @@
...
         <Claim>email,email_verified</Claim>
     </Scope>
-    <Scope id="profile">
+    <Scope id="profile" displayName="Profile" description="Retrieve profile information of the user.">
         <Claim>name,family_name,given_name,middle_name,nickname,preferred_username,profile,picture,website,gender,birthdate,zoneinfo,locale,updated_at
         </Claim>
     </Scope>
-    <Scope id="phone">
+    <Scope id="phone" displayName="Phone" description="View the phone number of the user.">
```
Comment (mefarazath) [inline features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml:31]: This returns the `phone_number_verified` as well. 
   So "View information related to phone number of the user." will be more appropriate?

Comment (vihanga-liyanage) [reply features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml:31]: +1

File: features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml (hunk 2 above)
Comment (vihanga-liyanage) [reply features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml]: In other places we do have used the .

File: features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <!--
-~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (mefarazath) [inline features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml:2]: o.O

**Review by mefarazath — APPROVED** [review]

File: features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml (hunk 5 above)
Comment (vihanga-liyanage) [reply features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml:2]: :p

**Pull request merged**